packages/bpk-component-drawer/src/BpkDrawer.tsx

```diff
@@ -63,13 +64,19 @@ const BpkDrawer = ({
   id,
   isIphone = isDeviceIphone(),
   isOpen,
+  mobileModalDisplay = false,
   onClose,
   padded = true,
   renderTarget = null,
   title,
 }: Props) =>  {
 
   const [isDrawerShown, setIsDrawerShown] = useState(true);
+  useEffect(() => {
```

Comment:
This replaces the deprecated UNSAFE_componentWillReceiveProps method from the .js implementation, without which isOpen and isDrawerShown fall out of sync and prevent the drawer being re-opened